File: PRAK603-2210817220011-MeldianaHandayani.c
#include <stdio.h>
int main(){
    int b1,b2,i,isi,hasil;
    scanf("%d %d",& b1,&b2);
    int num1[b1],num2[b2];
if(b1==b2){
    for(i=0;i<b1; i++ ){
        scanf("%d",&isi);
        num1[i]= isi;
    }
    for(i=0;i<b2; i++ ){
        scanf("%d",&isi);
        num2[i]= isi;
    }
    for(i=0;i<b1; i++ ){
        hasil=num1[i]*num2[i];
        printf("%d ", hasil);
    }
    }
    else{
        printf("Jumlah Tidak Sama ");

    }}
    

File: PRAK603-2210817220011-MeldianaHandayani.py
b1,b2=map(int,input().split())
if b1!=b2:
    print("Jumlah Tidak Sama")
else:
    n1=list(map(int,input().split()))
    n2=list(map(int,input().split()))
    m1=[]
    m2=[]
    z=0
    m1.append(n1[z:z+b1])
    m2.append(n2[z:z+b2])
    for i in range(0,b1):
        print(m1[0][i]*m2[0][i], end=' ')
    